api4/team_test.go

```diff
@@ -297,12 +297,12 @@ func TestUpdateTeam(t *testing.T) {
 		t.Fatal("Update failed")
 	}
 
-	team.Name = "Updated name"
+	team.Name = "newname23"
 	uteam, resp = Client.UpdateTeam(team)
 	CheckNoError(t, resp)
 
-	if uteam.Name == "Updated name" {
-		t.Fatal("Should not update name")
+	if uteam.Name != "newname23" {
```

Comment:
I'm concerned about this change of behaviour in the APIs. I've asked a Product Manager for guidance on whether this is OK or potentially problematic.

Reply:
What was the result of the discussion? Can we make this change?

Reply:
@hanzei @dchukmasov sorry this took a while to figure out. We feel that it's OK to change the `team.Name` field by CLI (as done here), but it shouldn't be possible to do so via the API (as this would require considerably additional client-side changes that are out of the scope of this PR). Please could you change the implementation to use `app.updateTeamUnsanitized()` instead of `app.UpdateTeam()`, so that `app.UpdateTeam()` doesn't need to be modified (and correspondingly the `api` unit tests don't change behaviour either)?

Reply:
@grundleborg I assume `app.updateTeamUnsanitized()` should have beenn used from the very beginning. Thank you for the reply, I've made the changes and updated the PR!